src/simulation/dynamics/constraintEffector/constraintDynamicEffector.cpp

```diff
@@ -120,6 +123,38 @@ void ConstraintDynamicEffector::setC_a(double c_a) {
     }
 }
 
+/*! This method allows the user to set the cut-off frequency of the low pass filter which is then used to calculate the coefficients for numerical low pass filtering based on a second-order low pass filter design.
+ @return void
+ @param wc The cut-off frequency of the low pass filter.
+*/
+void ConstraintDynamicEffector::setFilter_Data(double wc){
+    if (wc>0){
+    double k = 0.7;
```

Comment:
why are k and h hard coded?  This should be settable.  If `h` is the time step, then this needs to be settable as well as the tasks run at different rates.

Reply:
Agreed. Have updated this function to have h and k as settable parameters